lms/djangoapps/ccx/plugins.py

```diff
@@ -31,6 +34,14 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
+        if (
+            not isinstance(course.id, CCXLocator) and
```

Comment:
There is a function in the platform to check if the course is CCX, https://github.com/pearson-Advance/edx-platform/blob/pearson-release/juniper.master/common/djangoapps/student/auth.py#L36 @anfbermudezme